tasks/tangd-custom-port.yml

```diff
@@ -0,0 +1,59 @@
+---
+# This task tells SELinux for the port that we want the tangd
+# service to use when distribution Fedora
+- name: Allow the custom port for tangd_port_t in SELinux
+  import_role:
+    name: fedora.linux_system_roles.selinux
+  vars:
+    selinux_ports:
+      - ports: "{{ nbde_server_port }}"
+        proto: tcp
+        setype: tangd_port_t
+        state: present
+
+# This block creates the override file for systemd with the new
+# port that we have requested
+- name: Create override file
+  block:
+    # This task checks if the directory /etc/systemd/system/tangd.socket.d
+    # exist and registers the value in systemd__system__tangd_socket
+    - name: Check if directory /etc/systemd/system/tangd.socket.d exist
+      stat:
+        path: /etc/systemd/system/tangd.socket.d
+      register: systemd_system_tangd_socket
+
+    # This tasks Create the /etc/systemd/system/tangd.socket.d directory if
+    # it does not exist
+    - name: Create a directory if it does not exist
+      file:
+        path: /etc/systemd/system/tangd.socket.d
+        state: directory
+        mode: '0755'
+      when: systemd_system_tangd_socket.stat.exists
```

Comment:
We want to create the directory when it does not exist?
```
      when: not systemd_system_tangd_socket.stat.exists
```

Reply:
ops ... you are correct. I've fixed the problems and I'm pushing it right now.